packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -288,48 +264,168 @@ class CupertinoDialogAction extends StatelessWidget {
   }
 }
 
-const double _kButtonBarHeight = 45.0;
+/// Constructs a text content section typically used in a [CupertinoAlertDialog].
+///
+/// If Title is missing, then only message is added.  If Message is
```

Comment:
Wrap field references with []. Then it becomes linked in dart docs / API website / IDEs etc. e.g. `// If [title] is missing` etc...